waterloop/firmware-elesa
# CSS page for the Dashboard Operations page

- Created reusable styles (colours, fonts, and buttons)
- Set up the routing for the different pages
- created most of the operations page (some details need to be implemented but they can be done later on)

**Please read this styling ReadMe** `Frontend/src/js/styles/StylingOutline.md`

# How to test locally:
- pull the branch
- run `npm i`
- then run `npm start` to view the dashboard
- click through the links in the sidebar to ensure they are also working

Image for reference.
<img width="1675" alt="Screen Shot 2022-11-09 at 7 12 50 PM" src="https://user-images.githubusercontent.com/62915953/200969545-7e31e037-d903-48f1-9205-1e9954b08f4c.png">
